evalml/pipelines/components/estimators/regressors/exponential_smoothing_regressor.py

```diff
@@ -0,0 +1,130 @@
+"""Holt-Winters Exponential Smoothing Forecaster."""
+import numpy as np
+import pandas as pd
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+from evalml.utils import import_or_raise, infer_feature_types
+
+
+class ExponentialSmoothingRegressor(Estimator):
+    """Holt-Winters Exponential Smoothing Forecaster.
+
+    Currently ExponentialSmoothingRegressor isn't supported via conda install. It's recommended that it be installed via PyPI.
+
+    Args:
+        n_jobs (int or None): Non-negative integer describing level of parallelism used for pipelines. Defaults to -1.
+        random_seed (int): Seed for the random number generator. Defaults to 0.
+    """
+
+    name = "Exponential Smoothing Regressor"
+    hyperparameter_ranges = {}
+    """{
+    }"""
+    model_family = ModelFamily.EXPONENTIAL_SMOOTHING
+    """ModelFamily.EXPONENTIAL_SMOOTHING"""
+    supported_problem_types = [ProblemTypes.TIME_SERIES_REGRESSION]
+    """[ProblemTypes.TIME_SERIES_REGRESSION]"""
+
+    def __init__(
```

Comment:
Are these the only parameters we want to expose for `ExponentialSmoothingRegressor`? They're certainly not required as hyperparameters right now but `trend`, `damped_trend`, `seasonal`, `sp`, and `use_boxcox` could be good parameters to expose.